ambari-server/src/main/resources/common-services/HBASE/0.96.0.2.0/package/scripts/hbase_regionserver.py

```diff
@@ -51,6 +53,42 @@ def configure(self, env):
   def decommission(self, env):
     print "Decommission not yet implemented!"
 
+  def graceful_stop(self, env, upgrade_type=None):
```

Comment:
please consider moving implementations to `HBASE/0.96.0.2.0/package/scripts/hbase_service.py:26` (or, better, integrating them into existing start/stop code)

Reply:
Integrated with existing start/stop code. Please review